docs/en/latest/practices/check-crd-status.md

```diff
@@ -0,0 +1,102 @@
+---
+title: How to quickly check the synchronization status of CRD
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+When using the Apache APISIX Ingress Controller declarative configuration, often use the `kubectl apply` command. Only if the configuration was verified by its [Open API V3 Schema definition](https://swagger.io/specification/) and its validation webhooks (if any), can the configuration be accepted by Kubernetes.
+
+When the Apache APISIX Ingress Controller watches the resource change, the logic unit of the Apache APISIX Ingress Controller has just started to work. 
+In various operations of the Apache APISIX Ingress Controller, object conversion and more verification will be performed. 
+When the verification fails, the Apache APISIX Ingress Controller will throw an error message and will continue to retry 
+until the declared state is successfully synchronized to APISIX.
+
+Therefore, after the declarative configuration is accepted by Kubernetes, it does not mean that the configuration is synchronized to APISIX.
+
+In this practice, we will show how to  check the status of CRD.
+
+## Prerequisites
+
+- an available Kubernetes cluster
+- an available Apache APISIX and Apache APISIX Ingress Controller installation
+
+## Take ApisixRoute resource as an example 
+
+### deploy and check ApisixRoute resource
+
+1. first deploy an ApisixRoute resource
+
+e.g.
+   
+```yaml
+kubectl apply -f - <<EOF
+apiVersion: apisix.apache.org/v2alpha1
+kind: ApisixRoute
+metadata:
+  name: httpbin-route
+  namespace: test
```

Comment:
If namespace is specified here, you need to create it first.

Reply:
Namespapce removed.